v2/lints/etsi/lint_policy_and_qctype_are_consistent_test.go

```diff
@@ -0,0 +1,165 @@
+package etsi
+
+/*
+* ZLint Copyright 2020 Regents of the University of Michigan
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy
+* of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+* implied. See the License for the specific language governing
+* permissions and limitations under the License.
+ */
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v2/lint"
+	"github.com/zmap/zlint/v2/test"
+)
+
+func TestPolicyAndQCTypeConsistent(t *testing.T) {
+	testCases := []struct {
+		Name           string
+		InputFilename  string
+		ExpectedResult lint.LintStatus
+	}{
+		{
+			Name:           "evAllGood.pem",
+			InputFilename:  "evAllGood.pem",
+			ExpectedResult: lint.NA,
+		},
+		{
+			Name:           "QcStmtPsd2Cert80EuQcCertWoAnyPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert80EuQcCertWoAnyPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert81EuQcCertWoAnyPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert81EuQcCertWoAnyPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert82EuQcCertWoAnyPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert82EuQcCertWoAnyPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert83EuQcCertWoAnyPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert83EuQcCertWoAnyPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert84InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert84InconsistentPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert85InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert85InconsistentPolicy.pem",
```

Comment:
It wasn't clear to me how this differed from the test on 56, other than the QCT being different (eseal vs esign). It probably benefits from clearer file names, but since this does cover different code paths, I'm good here :)

Reply:
Done. Added comment to each certificate.